assignments/assignments.go

```diff
@@ -49,6 +52,13 @@ func UpdateFromTestsRepo(logger *zap.SugaredLogger, db database.Database, course
 		return
 	}
 	logger.Debugf("Assignments for %s successfully updated from '%s' repo", course.GetCode(), pb.TestsRepo)
+
+	ctx := context.Background()
+	err = handleTasks(ctx, db, scm, course, getTasksFromAssignments(ctx, assignments))
```

Comment:
Several things about `getTasksFromAssignments`:
- does not use/need `ctx`
- we should pass `assignments` into `handleTasks` and extract the tasks therein.
- that way we avoid calling the method here.